dbld/Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+def jobs = []
+
+if (!params.PLATFORMS)
+    jobs = ["centos-7", "debian-stretch", "debian-buster", "ubuntu-xenial", "ubuntu-bionic", "ubuntu-cosmic", "fedora-30"]
+else
+    jobs = params.PLATFORMS.split('\n')
+
+echo "Building packages for ${params.PLATFORMS}"
+
+def parallelStagesMap = jobs.collectEntries {
+    ["${it}" : generateStage(it)]
+}
+
+def generateStage(platform) {
+    return {
+        stage("${platform} - Copying Source") {
+            sh "rm -rf ${platform}"
+            sh "cp -ar syslog-ng ${platform}"
+        }
+        stage("${platform} - Docker Image") {
+            echo "Platform: ${platform}; Step: image"
+            sh "cd ${platform}; dbld/rules cache-image-${platform}"
+        }
+        stage("${platform} - Package") {
+            echo "Platform: ${platform}; Step: packaging"
+            sh "cd ${platform}; dbld/rules package-${platform}"
+            archiveArtifacts artifacts: "${platform}/dbld/build/${platform}/*"
+        }
+        stage("${platform} - kira") {
```

Comment:
Maybe this part (kira) is not needed for now. Later we can add the "kira" stage if we already tested/verified it.